src/Service/Startup.cs

```diff
@@ -429,6 +429,22 @@ public static LogLevel GetLogLevelBasedOnMode(RuntimeConfig runtimeConfig)
             return LogLevel.Error;
         }
 
+        /// <summary>
+        /// Takes in the RuntimeConfig object and checks the LogLevel level.
+        /// If LogLevel is not null, it will return the current value as a LogLevel,
+        /// else it will take the default option by going to GetLogLevelBasedOnMode
+        /// </summary>
+        public static LogLevel GetLogLevelBasedOnLevel(RuntimeConfig runtimeConfig)
```

Comment:
```suggestion
        public static LogLevel GetLogLevel(RuntimeConfig runtimeConfig)
```
nit: redundant name.

I added my suggestion. Another option, if you like better, is `GetConfiguredLogLevel()`

Reply:
How was this resolved? if you decide not to make any changes, please add follow up comment indicating which direction you chose to follow and your deciding factors.